addons/zeus/functions/fnc_bi_moduleCurator.sqf

```diff
@@ -240,7 +240,7 @@ if (_activated) then {
         _addons = [];
         {
             if (typeof _x == "ModuleCuratorAddAddons_F") then {
-                _paramAddons = call compile ("[" + (_x getvariable ["addons",""]) + "]");
+                _paramAddons = parseSimpleArray ("[" + (_x getvariable ["addons",""]) + "]");
```

Comment:
Optimising BI code, I like it 😆

Reply:
```
test getvariable ["addons",""]
"""ace_medical"", ""ace_zeus"""

call compile ("[" + (test getvariable ["addons",""]) + "]");
["ace_medical","ace_zeus"]

parseSimpleArray ("[" + (test getvariable ["addons",""]) + "]");
["ace_medical",<null>]
```
> Error Undefined variable in expression: _x 
 File z\ace\addons\zeus\functions\fnc_bi_moduleCurator.sqf, line 245

Reply:
The whitespace after comma 🤦‍♂️ 
I asked BI whether they'll remove that. See my comment below

Reply:
I'd say I revert the non 100% sure changes and we only leave the addAction changes from common?